### PR TITLE
[#4199] - Consolidating OGC URL Endpoints

### PR DESCRIPTION
**Pull request by jcalhoon96**
The changes here are to make the WMS and WCS endpoints match the WFS endpoint. The WCS endpoint is being changed because the current endpoint will generate a connection that pulls every resource with WCS compatible data into the GIS Client. The new endpoint which will match the WFS endpoint will fix this issue. The WMS endpoint is being changed to match the other two and be easier to manage.

Edit: I'm still new to updating and adding code to Github so the extra commits were just me testing to see how the changes after a pull request was made would work. Also I deleted the auto-generated checklist by accident.

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW VULNERABILITIES | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |
| NEW CODE SMELLS | 0 |
| NEW BUGS | 0 |
| NEW SECURITY HOTSPOTS | 0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4209](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4209)

**Comment by martinseul** [comment]
@sblack-usu could you please take a look.

File: hs_composite_resource/models.py [hunk 1]
```diff
@@ -586,11 +586,8 @@ def get_data_services_urls(self):
                     ] for data_type in resource_data_types
                 ):
                     wms_url = (
-                        f'{settings.HSWS_GEOSERVER_URL}/wms?'
```
Comment (sblack-usu) [inline hs_composite_resource/models.py:589]: Let's go ahead and simplify this whole function while we're in here.  The URL is formatted the same for all 3, it just uses wms, wfs, or wcs.  All of these if/else blocks are just confusing.  Below should be functionally the same.
```
    def get_data_services_urls(self):
        """
        Generates data services URLs for the resource.

        If the resource contains any GeoFeature or GeoRaster content, and if it's public,
        generate data service endpoints.
        """

        wfs_url = None
        wms_url = None
        wcs_url = None
        if self.raccess.public:
            try:
                resource_data_types = [lf.data_type for lf in self.logical_files]
                service_url = (
                        f'{settings.HSWS_GEOSERVER_URL}/HS-{self.short_id}/' +
                        '{}?request=GetCapabilities'
                    )
                if 'GeographicFeature' in resource_data_types:
                    wfs_url = service_url.format('wfs')
                    wms_url = service_url.format('wms')
                if 'GeographicRaster' in resource_data_types:
                    wcs_url = service_url.format('wcs')
                    wms_url = service_url.format('wms')

            except Exception as e:
                logger.exception("get_data_services_urls: " + str(e))

        data_services_urls = {
            'wms_url': wms_url,
            'wfs_url': wfs_url,
            'wcs_url': wcs_url
        }

        return data_services_urls
```

**Comment by jcalhoon96** [comment]
I just added the changes requested by Scott

**Pull request merged**